phpcs.xml.dist

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<ruleset name="PHP_CodeSniffer">
+    <description>The coding standard.</description>
+
+    <file>.</file>
+
+    <exclude-pattern>vendor/</exclude-pattern>
+
+    <!-- Show progress of the run -->
+    <arg value="p"/>
+
+    <arg name="extensions" value="php,module,inc,install,test,profile,theme,css,info,txt,yml"/>
+    <config name="drupal_core_version" value="9"/>
+
+    <rule ref="Drupal">
+        <!-- <exclude name="Drupal.Files.TxtFileLineLength.TooLong"/> -->
+        <!-- We want to be able to use "package" and "version" in our custom modules -->
+        <exclude name="Drupal.InfoFiles.AutoAddedKeys.Project"/>
+        <exclude name="Drupal.InfoFiles.AutoAddedKeys.Version"/>
+    </rule>
+
+    <rule ref="DrupalPractice"/>
```

Comment:
This is not a Drupal module. We could skip using code sniffer and only use PHP Cs Fixer.

Reply:
Removed